src/components/main-layout/ContentPanel.tsx

```diff
@@ -23,14 +20,34 @@ const ContentPanel: React.FC<ContentPanelProps> = ({
     callToAction,
     backButton,
     nextButton,
-    finishButton,
+    nextButtonText,
     moreInfo,
+    actionButton,
+    section,
+    currentModule,
 }) => {
-    const showButton = nextButton || finishButton;
-    const { exampleTrajectoryPageNumber, page } = useContext(SimulariumContext);
+    const showButton = nextButton;
+    const { page } = useContext(SimulariumContext);
+
+    const { totalMainContentPages } = useModule(currentModule);
```

Comment:
`App` unpacks this same data, I think it's possible that calling the hook again here could trigger unnecessary re-renders, even if the number that is returned is the same, my instinct is to just send `totalMainContentPages` down as a prop.